labgrid/resource/remote.py

```diff
@@ -207,6 +207,17 @@ def __attrs_post_init__(self):
         super().__attrs_post_init__()
 
 
+@target_factory.reg_resource
+@attr.s(eq=False)
+class NetworkYKUSHPowerPort(RemoteUSBResource):
+    """The NetworkYKUSHPowerPort describes a remotely YEPKIT YKUSH switchable USB hub"""
+    serial = attr.ib(default=None, validator=attr.validators.instance_of(str))
+    index = attr.ib(default=None, validator=attr.validators.instance_of(int))
```

Comment:
These should not have defaults. The current attrs library supports adding keyword-only arguments, so the default=None workaround shouldn't be necessary any more.